tags/tags.toml

```
@@ -220,7 +220,7 @@ keywords = [
     "abortcontroller",
 ]
 content = """
-Please update Node.js to version 16.11.0 or newer!
+Please update Node.js to the LTS version!
```

```suggestion
Please update Node.js to the [current LTS](https://nodejs.org/) (long-term support) version!
```
I think i prefer more verbosity here, with how many times i have to explain acronyms like LTS and ETA to people. It is always listed on the node org landing page, hence the link there seems useful.

---

Agreed.